android/settings.gradle

```diff
@@ -6,7 +6,8 @@ rootProject.name = 'native'
 include ':app'
 
 include ':backpack-react-native'
-project(':backpack-react-native').projectDir = new File(rootProject.projectDir, '../node_modules/backpack-react-native/android')
+project(':backpack-react-native').projectDir = new File(rootProject.projectDir, '../lib/android')
+project(':backpack-react-native').setBuildFileName('./build.internal.gradle')
```

Comment:
Added this so we can't publish a "clean" build.gradle file with the npm package and not worry about what we add to the internal one.